configs/configAVD2.py

```diff
@@ -7,32 +7,38 @@ class Config():
     """
 
     #Directories - MUST BE CHANGED for your environment
-    DATA_BASE_DIR = '/net/bvisionserver3/playpen/ammirato/sandbox/code/target_driven_instance_detection/Data/'
-    AVD_ROOT_DIR = '/net/bvisionserver3/playpen10/ammirato/Data/HalvedRohitData/'
+    # DATA_BASE_DIR = '/net/bvisionserver3/playpen/ammirato/sandbox/code/TDID/target_driven_instance_detection/Data/'
+    DATA_BASE_DIR = '/net/bvisionserver1/playmirror/mshvets/Phil_TDID/target_driven_instance_detection/Data/'
+    AVD_ROOT_DIR = '/net/bvisionserver3/playpen1/ammirato/Data/TDID_datasets/HalvedActiveVisionDataset/'
     FULL_MODEL_LOAD_DIR= os.path.join(DATA_BASE_DIR, 'Models/')
     SNAPSHOT_SAVE_DIR= os.path.join(DATA_BASE_DIR , 'Models/')
     META_SAVE_DIR = os.path.join(DATA_BASE_DIR, 'ModelsMeta/')
     TARGET_IMAGE_DIR= os.path.join(DATA_BASE_DIR, 'AVD_and_BigBIRD_targets_v1/')
     TEST_OUTPUT_DIR = os.path.join(DATA_BASE_DIR, 'TestOutputs/')
     TEST_GROUND_TRUTH_BOXES = os.path.join(DATA_BASE_DIR, 'GT/AVD_split2_test.json')
-    VAL_GROUND_TRUTH_BOXES = os.path.join(DATA_BASE_DIR ,'GT/AVD_part3_val.json')
+    VAL_GROUND_TRUTH_BOXES = os.path.join(DATA_BASE_DIR ,'GT/AVD_split2_test.json')
 
+    ATTENTION_ENABLED = True
+    # "sum", or "stack-reduce", or "attention-only"
+    ATTENTION_COMBINATION_MODE = "attention-only"
+    # "average" or "softmax"
+    ATTENTION_NORMALIZE_MODE = "average"
```

Comment:
I am adding these three flags. First is optional, as main code uses getattr and defaults to False. The other two are required in case first one is True.

Let's try to train a couple settings on the empty machines. Suggested:
1) sum + softmax
2) sum + average
3) attention-only + average
4) stack-reduce + softmax
5) stack-reduce + average

I don't believe too much in softmax version, upon our discussion